ci/aws/aws-cluster.lokocfg.envsubst

```diff
@@ -244,4 +244,9 @@ component "web-ui" {
     host                       = "web-ui.$CLUSTER_ID.$AWS_DNS_ZONE"
     certmanager_cluster_issuer = "letsencrypt-staging"
   }
+  oidc {
```

Comment:
We have changed the variable in docs. So I guess we should also change in CI from 

`DEX_STATIC_CLIENT_GANGWAY_ID` --> `DEX_STATIC_CLIENT_CLUSTERAUTH_ID`

and

`DEX_STATIC_CLIENT_GANGWAY_SECRET` -->
`DEX_STATIC_CLIENT_CLUSTERAUTH_SECRET`

For all platforms 
- aws, aws_edge and packet

Reply:
Yes, I marked that as "Pending work" in the commit message. Thanks for taking care of that!